state-chain/runtime/src/lib.rs

```diff
@@ -1600,6 +1600,39 @@ impl_runtime_apis! {
 					(asset, pallet_cf_lp::FreeBalances::<Runtime>::get(&account_id, asset).unwrap_or(0))
 				).collect(),
 				earned_fees: AssetMap::from_iter(pallet_cf_lp::HistoricalEarnedFees::<Runtime>::iter_prefix(&account_id)),
+				boost_balances: AssetMap::from_iter(Asset::all().map(|asset| {
+					let pool_details = Self::cf_boost_pool_details(asset);
+
+					(
+						asset,
+						pool_details.into_iter().filter_map(|(fee_tier, details)| {
+							let available_balance = details.available_amounts.into_iter().find_map(|(id, amount)| {
+								if account_id == id {
+									Some(amount)
+								} else {
+									None
+								}
+							})?;
```

Comment:
I think you might want to use `unwrap_or_default` here. Otherwise if the whole balance is tied up in pending boosts, BoostPoolInfo won't show up.

Reply:
I checked that case, it returns an available amount of `0`